packages/core/src/defer/timer_scheduler.ts

```diff
@@ -198,7 +200,11 @@ export class TimerScheduler {
 
         const timeout = Math.max(invokeAt - now, FRAME_DURATION_MS);
         this.invokeTimerAt = invokeAt;
-        this.timeoutId = setTimeout(callback, timeout) as unknown as number;
+        let timeoutId: number | null = null;
+        ngZone.runOutsideAngular(() => {
```

Comment:
```suggestion
       Const timeoutId = ngZone.runOutsideAngular(() => setTimeout…
```

suggestion: runOutsideAngular returns the result of the closure so you could simplify this a bit.

Reply:
I wondered about that. Thanks for the suggestion. I'll push the change.